app/views/people/by_task.html.erb

```diff
@@ -0,0 +1,38 @@
+<h2><%= @person.name %>'s Tasks</h2>
+<article>
+  <% if @tasks %>
+    <% @tasks.each do |task| %>
+      <section class="floating <%= task.status == 'Completed' ? 'completed' : '' %>">
+        <section class="card-head">
+          <strong> <h3><%= task.task %></h3></strong>
```

Comment:
Calling the value here `task` when your each/do is also using `task` as your singular is a bit confusing. If this is the task name, try renaming the variable to be `name` or `description` so that you're calling `task.name` or `task.description` for better readability.